api-get-bucket-encryption.go

```diff
@@ -0,0 +1,73 @@
+/*
+ * MinIO Go Library for Amazon S3 Compatible Cloud Storage
+ * Copyright 2020 MinIO, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package minio
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+
+	"github.com/minio/minio-go/v6/pkg/s3utils"
+)
+
+// GetBucketEncryption - get default encryption configuration for a bucket.
+func (c Client) GetBucketEncryption(bucketName string) (string, error) {
+	return c.GetBucketEncryptionWithContext(context.Background(), bucketName)
+}
+
+// GetBucketEncryptionWithContext gets the default encryption configuration on an existing bucket with a context to control cancellations and timeouts.
+func (c Client) GetBucketEncryptionWithContext(ctx context.Context, bucketName string) (string, error) {
```

Comment:
Just like Set, get should return a data structure

Reply:
ok

Reply:
ok